docs/UserGuide.md

```diff
@@ -55,26 +58,42 @@ This guide does not assume any technical knowledge from users. We just require u
 
 ## How to Use This Guide
 
-There is no wrong way to read this guide: if you want to skip ahead, go for it! Each section is contained in itself, and you can find what you are looking for without needing to read the previous sections. However, we recommend that you read through the entire guide at least once before utilizing the application, so that you would have a more thorough understanding of the all necessary features.
+There is no wrong way to read this guide: if you want to skip ahead, go for it! Each section is contained entirely in itself, and you can find what you are looking for without needing to read the previous sections. However, we recommend that you read through the entire guide at least once before utilizing the application, so that you would have a more thorough understanding of the all necessary features.
 
-This guide is comprised of 4 main sections:
-1. The [**Quick Start**](#quick-start) section, we will walk through you in how to get started installing as well as running the ConnectCare on your own desktop
+This guide has 4 main sections:
+1. The [**Quick Start**](#quick-start) section, we will walk through you in how to get started installing as well as running the ConnectCare on your own desktop.
 
 2. The [**Features Section**](#features) will provide a comprehensive, easy-to-understand guide on how to fully make use of all of ConnectCare's functionalities and commands.
 
 3. The [**FAQ**](#faq) section will answer commonly asked questions, addressing potential further queries you might have!
 
-4. Lastly, the [**Glossary**](#glossary) contains a list of key terms and definitions that might be helpful
+4. Lastly, the [**Glossary**](#glossary) contains a list of key terms and definitions that might be helpful.
+
+The various boxes used in this guide:
+<box type="tip">
+
+Gives you useful tips that are not entirely necessary to use the application, but help you get the most out of _ConnectCare_.
+</box>
+
+<box type="info">
+
+Provides additional information and context that extends your understanding of _ConnectCare_'s features.
+</box>
 
-<box type="warning" seamless>
+<box type="warning">
+
+Important warnings that you should keep in mind while using this User Guide.
```

Comment:
Is User Guide purposefully capitalized here?

Reply:
We can standardise this and always capitalise User Guide

Reply:
Hm, when I checked it seems as though it's always been capitalized?
![Screenshot 2024-03-26 130843](https://github.com/AY2324S2-CS2103T-W12-4/tp/assets/104015990/864eed7a-d5f1-4343-a953-0a886d5f6db5)

Reply:
I'll just resolve this for now